src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -0,0 +1,73 @@
+"""
+File       : MSRuleCleaner.py
+
+Description: MSRuleCleaner.py class provides the logic used to clean the Rucio
+block level data placement rules created by WMAgent.
+"""
+
+# futures
+from __future__ import division, print_function
+
+# system modules
+from retry import retry
+
+# WMCore modules
+from WMCore.MicroService.DataStructs.DefaultStructs import RULECLEANER_REPORT
+from WMCore.MicroService.Unified.MSCore import MSCore
+from Utils.EmailAlert import EmailAlert
+
+
+class MSRuleCleaner(MSCore):
+    """
+    MSRuleCleaner.py class provides the logic used to clean the Rucio
+    block level data placement rules created by WMAgent.
+    """
+
+    def __init__(self, msConfig, logger=None):
+        """
+        Runs the basic setup and initialization for the MSRuleCleaner module
+        :param msConfig: micro service configuration
+        """
+        super(MSRuleCleaner, self).__init__(msConfig, logger)
+
+        self.msConfig.setdefault("limitRequestsPerCycle", 500)
+        self.msConfig.setdefault("verbose", True)
+        self.msConfig.setdefault("interval", 60)
+        self.msConfig.setdefault("services", ['ruleCleaner'])
+        self.msConfig.setdefault("defaultGroup", "DataOps")
```

Comment:
I guess this configuration won't be needed, but we better x-check with Nick.

Reply:
I was actually planning to use it as a pointer to the Rucio group with which the block level rules were created. IIUC the one used  by the Agents should be different from the one used by for creating the container level groups. Or maybe it was just the user. Or maybe I am completely wrong though.